pkg/envoy/lds/ingress.go

```diff
@@ -57,27 +65,43 @@ func newIngressFilterChain(cfg configurator.Configurator, svc service.MeshServic
 	}
 }
 
-func getIngressFilterChains(svc service.MeshService, cfg configurator.Configurator) []*xds_listener.FilterChain {
+func (lb *listenerBuilder) getIngressFilterChains(svc service.MeshService) []*xds_listener.FilterChain {
```

Comment:
nit, unrelated to the commit, should it be `ForService`?

Reply:
Sure, fair enough.